python/dictionaries.md

```diff
@@ -1,12 +1,19 @@
 # Dictionaries
 
-You can think of lists as a _mapping_ from indices to values. The indices are 
-always integers and go from `0` to `len(the_list)`, and the values are the 
+You can think of lists as a _mapping_ from indices to values. The indices are
+always integers and go from `0` to `len(the_list)`, and the values are the
 items.
 
-Dictionaries are like lists, but they can map from more than just integers.  
-Because of this, you must always specify the index for every item when creating 
-a dictionary.
+Dictionaries are collections, just like lists, but they have important
+differences:
+
+* lists map sequential numeric indices to items, whereas dictionaries map
+  anything to anything,
```

Comment:
Technically not "anything", but the detail behind why is a bit gross 🤔

Reply:
Yes the detail actually follows...

Reply:
Indeed, so this creates a contradiction. Maybe "almost anything"?

Reply:
Let me rephrase ;-)